100_essentials/greetings/README.md

```diff
@@ -0,0 +1,59 @@
+# Greetings
+
+Everything you need to complete this challenge can be found [here](https://docs.python.org/3.8/tutorial/controlflow.html#defining-functions).
+
+---
+
+Define a function named `greet` that takes in a name and a message as inputs and returns a greeting in the following format:
+
+```python
+>>> greet("Mark", "Oh, hi")
+'Oh, hi, Mark!'
+```
+
+Additionally, the message should default to "Hello" if none is provided.
+
+```python
+>>> greet("Dave")
+'Hello, Dave!'
+```
```

Comment:
Related to the above (and something I didn't consider), are you having the greeting after the name because, as a parameter with a default argument, it's considered a keyword argument and therefore cant be placed in front of the non-default argument having parameter?